pkg/kubeapiserver/options/authentication_test.go

```diff
@@ -55,8 +56,12 @@ import (
 )
 
 func TestAuthenticationValidate(t *testing.T) {
+	testDataDir := filepath.Join("testdata")
```

Comment:
You can remove this variable and use "testdata" directly below. Linting is complaining about this bit

Reply:
Sure!